source/io/CsvReader.ooc

```diff
@@ -59,38 +62,35 @@ CsvReader: class extends Iterator<VectorList<Text>> {
 					case '\r' =>
 						continue
 					case '"' =>
-						textBuilder append(this _extractStringLiteral(row, i&))
+						stringBuilder add(this _extractStringLiteral(row, i&))
 					case =>
-						textBuilder append(readCharacter)
+						stringBuilder add(readCharacter)
 				}
 			}
-			result add(textBuilder toText())
-			textBuilder free()
+			result add(stringBuilder toString())
+			for (i in 0 .. stringBuilder count)
+				stringBuilder[i] free()
+			stringBuilder free()
```

Comment:
Or, the `StringBuilder` constructor should take an argument of whether it should free the content or not. That will be another PR though.